test/CMakeLists.txt

```diff
@@ -1458,7 +1458,10 @@ if(iodaconv_bufr_query_ENABLED)
                     COMMAND bash
                     ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
                             netcdf
-                            "${Python3_EXECUTABLE} testinput/prepbufr_adpsfc_api.py"
+                            "${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR}/src/ncepbufr/prepbufr_adpsfc_api.py
+                            -d 202108010000
+                            -i testinput/gdas.t12z.adpsfc.prepbufr
```

Comment:
this is most likely from `t12z` as indicated need to look at sample bufr to find *true* date

Reply:
I think I just used the input file that was already there